From c4b917456801770f9ec9fe1fd6043b6245f40ac0 Mon Sep 17 00:00:00 2001
From: Maarten Coene <maartenc@apache.org>
Date: Mon, 16 Nov 2009 22:49:17 +0000
Subject: [PATCH] Reverted change made for IVY-956 because the change for
 IVY-1115 also fixes the attached tests and is a cleaner solution...

git-svn-id: https://svn.apache.org/repos/asf/ant/ivy/core/trunk@881022 13f79535-47bb-0310-9956-ffa450edef68
---
 src/java/org/apache/ivy/core/resolve/VisitNode.java | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/src/java/org/apache/ivy/core/resolve/VisitNode.java b/src/java/org/apache/ivy/core/resolve/VisitNode.java
index 3937414d2..4a4d3e799 100644
--- a/src/java/org/apache/ivy/core/resolve/VisitNode.java
+++ b/src/java/org/apache/ivy/core/resolve/VisitNode.java
@@ -34,7 +34,6 @@
 import org.apache.ivy.core.resolve.IvyNodeEviction.EvictionData;
 import org.apache.ivy.plugins.conflict.ConflictManager;
 import org.apache.ivy.util.Checks;
-import org.apache.ivy.util.Message;
 
 /**
  * A visit node is an object used to represent one visit from one parent on an {@link IvyNode} of
@@ -276,12 +275,6 @@ public boolean loadData(String conf, boolean shouldBePublic) {
         if (loaded) {
             useRealNode();
             
-            // check if the real node is blacklisted -> if so, skip further loading
-            if (getRealNode().isBlacklisted(rootModuleConf)) {
-                Message.debug(rootModuleConf + " is blacklisted. Skip loading");
-                return false;
-            }
-            
             // if the loaded revision is different from original one
             // we now register this node on the new resolved id
             // this includes two cases:
